bot.py

```diff
@@ -1,23 +1,25 @@
-import discord, asyncio, os, platform, sys
+import discord
+import os
+import platform
+import sys
 from discord.ext.commands import Bot
-from discord.ext import commands
-if not os.path.isfile("config.py"):
-	sys.exit("'config.py' not found! Please add it and try again.")
-else:
+try:
 	import config
+except ImportError:
+	sys.exit("'config.py' not found! Please add it and try again.")
 
-
-intents = discord.Intents.default()
+intents = discord.Intents.all()
```

Comment:
This might require a change on the discord bot api website dashboard: This is a relatively new change by the discord bot api itself to be able to read the message's content.